corehq/apps/events/static/events/js/event_attendees.js

```diff
@@ -108,8 +136,76 @@ hqDefine("events/js/event_attendees",[
         return self;
     };
 
+    var newAttendeeCreationModel = function () {
+        var self = {};
+
+        // Make status constants available to bindings in HTML
+        self.STATUS_CSS = STATUS_CSS;
+
+        // Attendee in new attendee modal, not yet sent to server
+        self.stagedAttendee = ko.observable();
+
+        // New attendees sent to server
+        self.newAttendees = ko.observableArray();
+
+        self.allowSubmit = ko.computed(function () {
+            if (!self.stagedAttendee()) {
```

Comment:
Is optional chaining supported? If so, you could just chain these two statements and so something like the following:
```
if (self.stagedAttendee()?.name()) {
    return true;
}
return false;
```

If not you could combine the two statements with `if(!(self.stagedAttendee() && self.stagedAttendee().name()))`